arangod/Aql/ExecutionBlockImpl.cpp

```diff
@@ -1633,8 +1652,33 @@ ExecutionBlockImpl<Executor>::executeWithoutTrace(AqlCallStack stack) {
               clientCall.getLimit() == 0 && clientCall.needsFullCount();
 #endif
           LOG_QUERY("1f786", DEBUG) << printTypeInfo() << " call skipRows " << clientCall;
-          auto [state, stats, skippedLocal, call, dependency] =
-              executeSkipRowsRange(_lastRange, clientCall);
+
+          ExecutorState state = ExecutorState::HASMORE;
+          typename Executor::Stats stats;
+          size_t skippedLocal = 0;
+          AqlCall call{};
+          size_t dependency = 0;
+          if constexpr (is_one_of_v<Executor, SubqueryExecutor<true>>) {
+            // NOTE: The subquery Executor will by itself call EXECUTE on it's
```

Comment:
*Why* does it call execute itself?

Reply:
Because it calls the execute-machinery of the internal subquery branch.
This is unknown to the ExecutionBlockImpl.